patch/Patch.ts

```diff
@@ -0,0 +1,68 @@
+export const DEFAULT_ROW_DELIMITER = "..."
+export const DEFAULT_COLUMN_DELIMITER = "~"
+const DEFAULT_COLUMN_DELIMITER_ENCODED = "%7E"
+export class Patch {
+    object: any
+    uriEncodedString: string
+    private rowDelimiter: string
+    private columnDelimiter: string
+    constructor(
+        strOrObjLiteral: string | any = {},
```

Comment:
I'm wondering if we should narrow down that `any` to something like `Record<string, any>`, or even `Record<string, string | number>` if that's accurate?

Reply:
Yes! On it